.github/workflows/build-openstack-capi-image.yml

```diff
@@ -0,0 +1,85 @@
+name: Build OpenStack VM CAPI image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Kubernetes version
+        required: true
+        type: string
+      tag:
+        description: ck8s-capi tag
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        description: Kubernetes version
+        required: true
+        type: string
+      tag:
+        description: ck8s-capi tag
+        required: true
+        type: string
+
+env:
+  version: ${{ inputs.version }}
+  tag: ${{ inputs.tag }}
+
+
+defaults:
+  run:
+    working-directory: ./images/capi
+
+jobs:
+  build-image:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v5
+        name: Checkout
+
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/.config/packer/plugins
+          key: packer-${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('**/*.pkr.hcl') }}
+              
+      - name: Enable KVM
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
+
+      - name: install qemu-kvn
+        run: |
+          sudo apt update && \
+          sudo apt upgrade -y && \
+          sudo apt install -y qemu-kvm libvirt-daemon-system libvirt-clients bridge-utils qemu-utils
```

Comment:
Question: Could we not build an image that has these things already and use that? Would save some time and be less error-prone